data/mods/blazemod/blaze_blob_parts.json

```diff
@@ -15,7 +15,20 @@
     "damage_modifier": 10,
     "folded_volume": 1,
     "breaks_into": [ { "item": "bitergrow" } ],
-    "flags": [ "TURRET", "TOOL_NONE", "FOLDABLE", "OBSTACLE" ]
+    	"requirements": {
```

Comment:
This should use spaces instead of tabs, and line up with "install" on the next line.